pkg/daemon/ts2phcconfig.go

```diff
@@ -0,0 +1,88 @@
+package daemon
```

Comment:
can you name this ts2phc_config.go instead, to be more consistent with the ptp operator?

Reply:
actually, is there any reason this needs to be in a separate file than config.go?

Reply:
just to keep those things separate ,since we don't know what else are discovered in this process of POC.
Yes I will rename the file.

Reply:
Fixed